lib/storage-handler.js

```diff
@@ -253,6 +253,7 @@ exports.download = function(provider, req, res, container, file, cb) {
     reader.pipe(res);
 
     reader.on('error', function onReaderError(err) {
+      res.set('content-type', 'application/json');
```

Comment:
@bajtos 
it calls `res.type` https://github.com/strongloop/loopback-component-storage/blob/1.x/lib/storage-handler.js#L251 and never changes again, I believe because it is a download method, it does not perform a content-negotiation. but when has an error, it keeps such `content-type`

**lb 2.x**
then in rest-adapter [strong-remoting@`2.x`](https://github.com/strongloop/strong-remoting/blob/v2.32.3/lib/rest-adapter.js#L379-L400) it only does a `res.send()`

**lb 3.x**
in 3.x because the errorHandler defaults to sendJson(which also sets content-type) its not a problem https://github.com/strongloop/strong-error-handler/blob/f1d5f064f01fc3c00c15dc62cbde31a1767a77bd/lib/content-negotiation.js#L77-L85